tmt/steps/__init__.py

```diff
@@ -117,10 +147,12 @@ def plugins_in_standalone_mode(self):
         return sum(phase.is_in_standalone_mode for phase in self.phases())
 
     @classmethod
-    def usage(cls, method_overview):
+    def usage(cls, method_overview: str) -> str:
         """ Prepare general usage message for the step """
         # Main description comes from the class docstring
+        assert cls.__doc__ is not None
```

Comment:
Nice, helpful exception would be better here, ugly `AssertionError` is hard to read for users.

Reply:
Right, added conditional with raising fot both `cls.__doc__ ` and `cls.__name__ `